distributionviewer/api/views.py

```diff
@@ -32,8 +32,8 @@ def render_numeric(dist):
     return s.data
 
 
+@login_required
```

Comment:
Since you've already updated the default DRF settings to include `IsAuthenticated`, is this needed?